docs/en/sql-reference/aggregate-functions/reference/quantileApprox.md

```diff
@@ -0,0 +1,76 @@
+---
+slug: /en/sql-reference/aggregate-functions/reference/quantileApprox
+sidebar_position: 204
+---
+
+# quantileApprox
```

Comment:
Almost the same as in `quantiles.md`

Reply:
Because quantileApprox and quantilesApprox are quite similar. How do you advice to improve it ?

Reply:
In `quantilesApprox`, we can refer to `quantileApprox` and say that `quantilesApprox` works similarly but allows us to calculate quantities at different levels simultaneously and returns a tuple, and add only new examples.

Reply:
ok.